### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
All of these except for the `<html lang="en"` ones are to provide higher contrast. The fact that Bootstrap classes don't have sufficient contrast by default with a white background is a known thing that they don't seem to have plans to fix.

Something I learned as a result of this is that IdentityUI has it's own layout and styling which don't seem to integrate with the SPA frameworks (because they're Razor). The result is that changes to SPA templates are not reflected in IdentityUI pages at all. For example if you were to make the background of your SPA site black instead of white then go to the login page, the background of the login page would still be  white. That seems less than ideal.

### CODE REVIEW

**Review by Eilon — APPROVED** [review]
Looks alright to me, but I don't know this code very well. One of @javiercn or @HaoK should review as well.

**Comment by javiercn** [comment]
Can you post images with the visual changes?

**Comment by javiercn** [comment]
There isn’t anything we can do about the SPA/Identity layout impedance as they are built with completely different technologies.

**Comment by ryanbrandenburg** [comment]
countercontrast
contrast

Here are some examples from the Angular template. Basically the offending elements were just made a couple shades darker.

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ import ApiAuthorizationRoutes from './components/api-authorization/ApiAuthorizat
 import { ApplicationPaths } from './components/api-authorization/ApiAuthorizationConstants';
 ////#endif
 
+import './custom.css'
```
Comment (javiercn) [inline src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js]: Don’t name this custom. Look up the common react convention for this and if not either name it site.css or app.css to match the file.

Comment (javiercn) [reply src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js]: Wait, seems like custom was what we were doing. Why do we change it from scss to css (as I expect scss to be processed at build)

**Review by javiercn — APPROVED** [review]
Looks good, I want to better understand the move from scss to css.

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js (hunk 1 above)
Comment (ryanbrandenburg) [reply src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js]: The scss file that existed before was both empty and unused. If someone had tried to import it as I do here they would have gotten an error like `You attempted to import ../scss/custom.scss which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.`. If we try just moving the scss file within the src directory we instead get `To import Sass files, you first need to install node-sass.
Run `npm install node-sass` or `yarn add node-sass` inside your workspace.`. I figured that it wasn't worth it to add an additional OSS dependency when the styles we're applying makes no use of the additional features from scss, so I made it a css file.

**Comment by ryanbrandenburg** [comment]
@aspnet-hell triage https://dev.azure.com/dnceng/public/_build/results?buildId=192570

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2535

**Comment by ryanbrandenburg** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=192575

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2432
https://github.com/aspnet/AspNetCore-Internal/issues/2445
https://github.com/aspnet/AspNetCore-Internal/issues/2481

**Pull request merged**